www-client/uget/uget-2.2.0.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit gnome2-utils xdg-utils
+
+IUSE="+gnutls aria2 gstreamer appindicator libnotify nls openssl sockets rss"
+KEYWORDS="~amd64 ~arm ~x86"
+SRC_URI="mirror://sourceforge/urlget/${P}.tar.gz"
+
+DESCRIPTION="Download manager using gtk+ and libcurl"
+HOMEPAGE="http://www.ugetdm.com"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+
+DOCS=( AUTHORS ChangeLog NEWS README )
+
+REQUIRED_USE="^^ ( gnutls openssl )"
+RDEPEND="
+	>=net-misc/curl-7.19.1
+	dev-libs/libpcre
+	>=dev-libs/glib-2.32:2
+	>=x11-libs/gtk+-3.4:3
+	gnutls? ( net-libs/gnutls dev-libs/libgcrypt:= )
```

Comment:
Slot `:11` doesn't provide headers, so you want `:0=`. Also please don't list multiple deps in a single line.